eq-author/package.json

```diff
@@ -66,26 +66,26 @@
     "eslint": "^5.6.1",
     "eslint-config-eq-author": "^2.0.1",
     "eslint-import-resolver-webpack": "^0.8.4",
-    "eslint-loader": "2.1.1",
+    "eslint-loader": "^2.1.1",
     "eslint-plugin-graphql": "^1.5.0",
-    "file-loader": "2.0.0",
-    "fs-extra": "7.0.0",
+    "file-loader": "^2.0.0",
+    "fs-extra": "^7.0.0",
     "gh-pages": "^1.0.0",
     "gzip-size": "^4.0.0",
-    "html-webpack-plugin": "4.0.0-alpha.2",
+    "html-webpack-plugin": "^4.0.0-alpha.2",
```

Comment:
Should we be depending on an alpha release?

Reply:
It is required for webpack 4. Bit rubbish, but there is now a beta which this resolves to in the `yarn.lock` file. Also I have a greater tolerance for alpha/beta build tools as our tests run on the output.